backend/src/main/java/com/zeros/backend/RestaurantResource.java

```diff
@@ -18,4 +18,9 @@
     public List<Restaurant> getRestaurants() {
         return restaurantRecommender.getNearbyRestaurant();
     }
+
+    @GetMapping("/increment/{restaurantId}")
+    public void incrementRestaurantCounter( String restaurantId ) {
+        restaurantRecommender.incrementRestaurantCounter(restaurantId);
+    }
```

Comment:
restaurantRecommender it's not his responsibility to increment restaurant counter

Reply:
LoyaltyRewarder ??